### PR TITLE
Test fix

### PR DESCRIPTION
**Pull request by echon006**
Cleaned up stat tracked and fixed winning/worst coach

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -202,32 +163,23 @@ def rival(team_id)
...
+    game_ids = @games_mngr.game_ids_in_games(@games_mngr.games_in_season(season))
+    @gt_mngr.percentage_wins_by_coach(game_ids).max_by { |coach , average_wins| average_wins }[0]
   end
 
   def worst_coach(season)
-    average_wins_by_coach(season).min_by { |coach , average_wins| average_wins }[0]
+    game_ids = @games_mngr.game_ids_in_games(@games_mngr.games_in_season(season))
+    @gt_mngr.percentage_wins_by_coach(game_ids).min_by { |coach , average_wins| average_wins }[0]
   end
+  # def game_teams_by_games(games)
```
Comment (LelandCurtis) [inline lib/stat_tracker.rb:174]: uncomment

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -202,32 +163,23 @@ def rival(team_id)
...
+  #    @gt_mngr.game_teams.find_all{|game_team|game_ids.include?(game_team.game_id)}
+  #  end
 
-  # accept an array of game teams and return a single accuracy score
-  # def accuracy(game_teams)
-  #   total_goals = game_teams.map{|game_team| game_team.goals}.sum.to_f
-  #   total_shots = game_teams.map{|game_team| game_team.shots}.sum.to_f
-  #   accuracy = total_goals/total_shots
-  # end
+  #  def game_teams_in_season(season)
```
Comment (LelandCurtis) [inline lib/stat_tracker.rb:179]: uncomment

**Review by kevingloss — APPROVED** [review]
Code looks good, and spacing is looking how we want.

**Review by LelandCurtis — APPROVED** [review]
looks good.

**Pull request merged**